cardano-db-sync/src/Cardano/DbSync/LocalStateQuery.hs

```diff
@@ -52,12 +52,16 @@ import Ouroboros.Network.Protocol.LocalStateQuery.Client (
  )
 import qualified Ouroboros.Network.Protocol.LocalStateQuery.Client as StateQuery
 import Ouroboros.Network.Protocol.LocalStateQuery.Type (AcquireFailure)
+import qualified Cardano.Ledger.BaseTypes as Ledger
+import qualified Ouroboros.Consensus.Node.ProtocolInfo as Consensus
 
-data NoLedgerStateEnv = NoLedgerStateEnv
-  { nlsTracer :: Trace IO Text
-  , nlsSystemStart :: !SystemStart
-  , nlsQueryVar :: StateQueryTMVar CardanoBlock CardanoInterpreter
-  , nlsHistoryInterpreterVar :: StrictTVar IO (Strict.Maybe CardanoInterpreter)
+data NoLedgerEnv = NoLedgerEnv
+  { nleTracer :: Trace IO Text
+  , nleSystemStart :: !SystemStart
+  , nleQueryVar :: StateQueryTMVar CardanoBlock CardanoInterpreter
+  , nleHistoryInterpreterVar :: StrictTVar IO (Strict.Maybe CardanoInterpreter)
+  , nleNetwork :: !Ledger.Network
+  , nleProtocolInfo :: !(Consensus.ProtocolInfo IO CardanoBlock)
```

Comment:
The addition of these two values, I hope is ok? was trying to make it match the parts that are shared in `HasLedgerEnv`